anomaly_detector/anomaly_detector.py

```diff
@@ -215,11 +222,28 @@ def infer(self):
         # When we reached # of inference loops, retrain models
         self.recreate_models = False
 
+    def fetch_false_positives(self):
+        """Fetch false positive from datastore and add noise to training run."""
+        _LOGGER.info("Fetching false positives from fact store")
+        try:
+            r = requests.get(url=self.config.FACT_STORE_URL + "/api/false_positive")
```

Comment:
It would be good to check StatusCode from the response

Reply:
We will be moving this to GRPC so the rest http stuff is going to be removed in future PR. I will open a git issue for now.